### ISSUE TITLE
Slider counter

### ISSUE THREAD

**Issue by bombarda-rpd** [CrocoBlock/suggestions#7600]
<strike>It will be nice to have an option to add a slider counter (like slide 1 of 5) on the slider widget.</strike>

- [ ] controls for moving fractions pagination (above the image or inside the slider)
- [x] change the separator and ability to add text before and after fraction pagination

**Comment by Crocoblock** [comment]
Hi @bombarda-rpd 

You can use this option (`Use Fraction pagination?`)
![image](https://github.com/Crocoblock/suggestions/assets/47778274/9ea8f02c-cc5f-4076-912a-7e945d4cba78)

**Comment by bombarda-rpd** [comment]
Yes, I found it, but some options are missing, such as the position of the fraction numbers. It might be possible to place it above the image (inside the slider)

**Comment by aalexandr21** [comment]
was added and released in JetElements 2.7.9
